fandogh_cli/fandogh_client/__init__.py

```diff
@@ -308,6 +311,16 @@ def deploy_manifest(manifest):
                              headers={'Authorization': 'JWT ' + token}
                              )
     if response.status_code != 200:
+        if response.status_code == 400:
+            keys = str(list(response.json().keys())[0]).split('.')
+            temp = x
```

Comment:
what is x?
please Improve variable namings

Reply:
+1